config.schema.json

```diff
@@ -20,11 +23,38 @@
             "items": {
                 "$ref": "#/$defs/featured_market"
             }
+        },
+        "blacklisted_markets": {
+            "type": "array",
+            "items": {
+                "$ref": "#/$defs/blacklisted_markets"
+            }
+        },
+        "blacklisted_pools": {
+            "type": "array",
+            "items": {
+                "$ref": "#/$defs/blacklisted_pools"
+            }
+        },
+        "blacklisted_tokens": {
+            "type": "array",
+            "items": {
+                "$ref": "#/$defs/blacklisted_tokens"
```

Comment:
Can make the struct id singular?
i.e.
`"...blacklisted_tokens"` => `"...blacklisted_token"`